fbfmaproom/fbf-update-data.py

```diff
@@ -33,7 +33,27 @@
     ),
     (
         "madagascar/enacts-precip-djf",
-        "http://iridl.ldeo.columbia.edu/home/.rijaf/.Madagascar_v3/.ALL/.monthly/.rainfall/.rfe/T/(Dec-Feb)/seasonalAverage//units/(mm/month)/def/",
+        "http://map.meteomadagascar.mg/SOURCES/.Madagascar_v4/.ALL/.monthly/.rainfall/.rfe/T/(Dec-Feb)/seasonalAverage//units/(mm/month)/def/X/42.525/48.975/RANGE/Y/-25.9875/-20.025/RANGE/",
```

Comment:
Was this slow to download? It (and the DJF version) would be faster if we did the spatial subsetting before the seasonalAverage. But if it's already fast, no point going back and testing again.

Reply:
ditto for the following one.

Reply:
The only one that was slow was NDVI which is just inherently slow so I'm leaving it as is